x-pack/plugins/cases/public/common/mock/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export * from './test_providers';
```

Comment:
What about `match_media` and `kibana_react.mock.ts` (if needed. See my comment below)

Reply:
there's no modules exported from match_media. it is a window mock, all you need to do is import like `import '../../common/mock/match_media';`